### DISCUSSION TITLE
Some resources for future studies

### DISCUSSION THREAD

**Discussion by FanwangM** [theochem/Selector#113] [Ideas]
The following list might be interesting to explore,

- https://arxiv.org/pdf/2110.09697.pdf and https://github.com/Mamba413/bess
- https://dl.acm.org/doi/pdf/10.1145/3462187
- https://pymoo.org/customization/subset.html
- https://github.com/decile-team/cords/

**Comment by PaulWAyers** [comment]
Interesting. These also deal with the same problem. On brief perusal, it isn't clear (to me) how different. It seems for some of these subsets are partly biased by the method (e.g., a linear model) which allows the optimal diversity to be selected effectively. Our approaches are more agnostic, and are based on (inherent) diversity. We should, nonetheless, include these in a comparison @Shania99 to the extent it is possible.

**Reply by PaulWAyers** [reply]
@xychem you should look at this discussion post.

**Reply by FanwangM** [reply]
Please feel free to tag me with any questions you may have. @xychem
